modules/nf-core/spaceranger/count/main.nf

```diff
@@ -31,17 +31,23 @@ process SPACERANGER_COUNT {
     def cytaimage = cytaimage ? "--cytaimage=\"${cytaimage}\"" : ""
     def darkimage = darkimage ? "--darkimage=\"${darkimage}\"" : ""
     def colorizedimage = colorizedimage ? "--colorizedimage=\"${colorizedimage}\"" : ""
+    def slide = meta.slide ? meta.slide : ""
+    def area = meta.area ? meta.area : ""
+    if (slide.matches("visium-(.*)") && area == "" && slidefile == "") {
+        slide_and_area = "--unknown-slide=\"${slide}\""
+    } else {
+        slide_and_area = "--slide=\"${slide}\" --area=\"${area}\""
+    }
```

Comment:
Do we need this at all? We could also rely on the pipeline developers to specify `--area` etc. via `task.ext.args`, e.g. 

```nextflow
process {
  withName: SPACERANGER_COUNT {
     task.ext.args = { (meta.slide & meta.area) ? "--area=\"${meta.area}\" --slide=\"${meta.slide}\"" : "--unknown-slide" }
  }
```
}

Reply:
Yes, I did think about this, but since this is _specifically_ part of the Space Ranger Count software as an explicit option, I thought it made more sense to have it in the module itself. It's also required to actually specify the version of Visium being run (_e.g._ `--unknown-slide visium-1`), which is also a demand of the software itself. It felt odd to me to write something so software-specific outside of the software module.